src/tr1/game/objects/creatures/ape.c

```diff
@@ -126,7 +126,9 @@ void Ape_Setup(OBJECT *obj)
     obj->save_hitpoints = 1;
     obj->save_anim = 1;
     obj->save_flags = 1;
-    g_AnimBones[obj->bone_idx + 52] |= BF_ROT_Y;
+
+    ANIM_BONE *const bone = (ANIM_BONE *)&g_AnimBones[obj->bone_idx];
+    (&bone[13])->rot_y = 1;
```

Comment:
Hi, we can reduce the ASCII spaghetti by using just `bone[13].rot_y = 1;`.

Reply:
Thanks, done.